static/app/utils/fields/index.ts

```diff
@@ -2494,6 +2495,80 @@ const SPAN_FIELD_DEFINITIONS: Record<string, FieldDefinition> = {
 
 const LOG_FIELD_DEFINITIONS: Record<string, FieldDefinition> = {
   ...LOG_AGGREGATION_FIELDS,
+  ...EVENT_FIELD_DEFINITIONS,
+  [OurLogKnownFieldKey.CODE_FILE_PATH]: {
+    desc: t(
+      'The source code file name that identifies the code unit as uniquely as possible (preferably an absolute file path).'
+    ),
+    kind: FieldKind.FIELD,
+    valueType: FieldValueType.STRING,
+  },
+  [OurLogKnownFieldKey.CODE_LINE_NUMBER]: {
+    desc: t(
+      'The line number in code.filepath best representing the operation. It SHOULD point within the code unit named in code.function'
```

Comment:
Nit, could you pull out `code.filepath` as a variable, in case there's a world where we ever change this key.

Reply:
Can do 👍